src/utils/github.ts

```diff
@@ -94,15 +94,21 @@ export class GithubApi {
   /**
    * Get content from the github repository
    */
-  async getContent(path: string): Promise<string> {
+  async getContent(path: string): Promise<string | null> {
     logger.info({ path }, 'GitHub API: Get Content');
-    const response = await this.octokit.rest.repos.getContent({ owner: this.owner, repo: this.repo, path });
-    if (!this.isOk(response.status)) throw new Error('Failed to get aerial TileSet config.');
-    if ('content' in response.data) {
-      return Buffer.from(response.data.content, 'base64').toString();
-    } else {
-      throw new Error(`Unable to find the content from path ${path}.`);
+    try {
+      const response = await this.octokit.rest.repos.getContent({ owner: this.owner, repo: this.repo, path });
+      if (this.isOk(response.status)) {
+        if ('content' in response.data) {
+          return Buffer.from(response.data.content, 'base64').toString();
+        } else {
+          throw new Error(`Unable to find the content from path ${path}.`);
+        }
+      }
+    } catch {
+      logger.debug({ path }, 'GitHub: Content not found');
     }
+    return null;
```

Comment:
Don't we want to throw the exception all the way out if there's a problem? Then we won't have to special-case `null` handling in callers.

Reply:
The way I am trying to handling is to not error out the code from failure response here, and only return null if any errors, so we treat all error as no content found from the github.

Reply:
Returning `null` in case of any error means we lose information, which means we can't debug without changing the code.

Reply:
I have updated this to return null only when a 404 status been catched.

Reply:
But now the code ignores all other errors. You probably want a `throw error` on line 116.

Reply:
Good point, just added that.